### ISSUE TITLE
Uninstall.bat fails

### ISSUE THREAD

**Issue by JackofSpades707** [mintty/wsltty#263]
Steps to reproduce this behavior: 
* open cmd.exe as admin
* cd %LOCALAPPDATA%\wsltty
* uninstall.bat

Expected behavior: 
* Completely uninstalls wsltty

Actual behavior:
* Messed up wsltty install left on system, I can open wsltty as admin as see that the prompt for my shell is now [messed up](https://i.imgur.com/6LsSHkF.png)

output :
```
C:\Users\jacko\AppData\Local\wsltty>comp/M "add default to context menu.lnk" "C:\Users\jacko\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\add default to context menu.lnk"
Comparing add default to context menu.lnk and C:\Users\jacko\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\add default to context menu.lnk...
Can't find/open file: C:\Users\jacko\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\add default to context menu.lnk


C:\Users\jacko\AppData\Local\wsltty>comp/M "add to context menu.lnk" "C:\Users\jacko\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\add to context menu.lnk"
Comparing add to context menu.lnk and C:\Users\jacko\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\add to context menu.lnk...
Can't find/open file: C:\Users\jacko\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\add to context menu.lnk


C:\Users\jacko\AppData\Local\wsltty>comp/M "configure WSL shortcuts.lnk" "C:\Users\jacko\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\configure WSL shortcuts.lnk"
Comparing configure WSL shortcuts.lnk and C:\Users\jacko\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\configure WSL shortcuts.lnk...
Can't find/open file: C:\Users\jacko\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\configure WSL shortcuts.lnk


C:\Users\jacko\AppData\Local\wsltty>comp/M "remove from context menu.lnk" "C:\Users\jacko\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\remove from context menu.lnk"
Comparing remove from context menu.lnk and C:\Users\jacko\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\remove from context menu.lnk...
Can't find/open file: C:\Users\jacko\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\remove from context menu.lnk


C:\Users\jacko\AppData\Local\wsltty>comp/M "cmd2.bat" "C:\Users\jacko\AppData\Local\Microsoft\WindowsApps\cmd2.bat"
Comparing cmd2.bat and C:\Users\jacko\AppData\Local\Microsoft\WindowsApps\cmd2.bat...
Can't find/open file: C:\Users\jacko\AppData\Local\Microsoft\WindowsApps\cmd2.bat


C:\Users\jacko\AppData\Local\wsltty>comp/M "dequote.bat" "C:\Users\jacko\AppData\Local\Microsoft\WindowsApps\dequote.bat"
Comparing dequote.bat and C:\Users\jacko\AppData\Local\Microsoft\WindowsApps\dequote.bat...
Can't find/open file: C:\Users\jacko\AppData\Local\Microsoft\WindowsApps\dequote.bat


C:\Users\jacko\AppData\Local\wsltty>comp/M "uninstall.bat" "C:\Users\jacko\AppData\Local\Microsoft\WindowsApps\uninstall.bat"
Comparing uninstall.bat and C:\Users\jacko\AppData\Local\Microsoft\WindowsApps\uninstall.bat...
Can't find/open file: C:\Users\jacko\AppData\Local\Microsoft\WindowsApps\uninstall.bat

Active code page: 65001
distro 'docker-desktop-data'
- name 'docker-desktop-data'
- guid {0c5ddd50-70f6-4dd8-98d6-2a8e16088bdd}
- (launcher C:\Users\jacko\AppData\Local/Microsoft/WindowsApps/docker-desktop-data.exe)
- icon C:\Users\jacko\AppData\Local\wsltty\wsl.ico
- root \\?\C:\Users\jacko\AppData\Local\Docker\wsl\data/rootfs
Setting +x wslbridge2 backends for distro 'docker-desktop-data'

C:\Users\jacko\AppData\Local\wsltty>del "/F" "C:\Users\jacko\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\docker-desktop-data Terminal.lnk"
Could Not Find C:\/F
Could Not Find C:\Users\jacko\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\docker-desktop-data Terminal.lnk

C:\Users\jacko\AppData\Local\wsltty>del "/F" "C:\Users\jacko\AppData\Local\Microsoft\WindowsApps\docker-desktop-data.bat"
Could Not Find C:\/F
Could Not Find C:\Users\jacko\AppData\Local\Microsoft\WindowsApps\docker-desktop-data.bat

C:\Users\jacko\AppData\Local\wsltty>del "/F" "C:\Users\jacko\AppData\Local\Microsoft\WindowsApps\docker-desktop-data~.bat"
Could Not Find C:\/F
Could Not Find C:\Users\jacko\AppData\Local\Microsoft\WindowsApps\docker-desktop-data~.bat
distro 'docker-desktop'
- name 'docker-desktop'
- guid {15685653-ca24-4ca0-82fe-22b941683b0e}
- (launcher C:\Users\jacko\AppData\Local/Microsoft/WindowsApps/docker-desktop.exe)
- icon C:\Users\jacko\AppData\Local\wsltty\wsl.ico
- root \\?\C:\Users\jacko\AppData\Local\Docker\wsl\distro/rootfs
Setting +x wslbridge2 backends for distro 'docker-desktop'

C:\Users\jacko\AppData\Local\wsltty>del "/F" "C:\Users\jacko\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\docker-desktop Terminal.lnk"
Could Not Find C:\/F
Could Not Find C:\Users\jacko\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\docker-desktop Terminal.lnk

C:\Users\jacko\AppData\Local\wsltty>del "/F" "C:\Users\jacko\AppData\Local\Microsoft\WindowsApps\docker-desktop.bat"
Could Not Find C:\/F
Could Not Find C:\Users\jacko\AppData\Local\Microsoft\WindowsApps\docker-desktop.bat

C:\Users\jacko\AppData\Local\wsltty>del "/F" "C:\Users\jacko\AppData\Local\Microsoft\WindowsApps\docker-desktop~.bat"
Could Not Find C:\/F
Could Not Find C:\Users\jacko\AppData\Local\Microsoft\WindowsApps\docker-desktop~.bat
distro 'Arch'
- name 'Arch'
- guid {59953fab-a692-4f2b-a872-d1706af526db}
- (launcher C:\Users\jacko\AppData\Local/Microsoft/WindowsApps/Arch.exe)
- icon C:\Users\jacko\AppData\Local\wsltty\wsl.ico
- root C:\Users\jacko\scoop\persist\archwsl\data/rootfs
Setting +x wslbridge2 backends for distro 'Arch'

C:\Users\jacko\AppData\Local\wsltty>del "/F" "C:\Users\jacko\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Arch Terminal.lnk"
Could Not Find C:\/F

C:\Users\jacko\AppData\Local\wsltty>del "/F" "C:\Users\jacko\AppData\Local\Microsoft\WindowsApps\Arch.bat"
Could Not Find C:\/F

C:\Users\jacko\AppData\Local\wsltty>del "/F" "C:\Users\jacko\AppData\Local\Microsoft\WindowsApps\Arch~.bat"
Could Not Find C:\/F
distro ''
- name 'WSL'
- guid DefaultDistribution
- (launcher C:\Users\jacko\AppData\Local/Microsoft/WindowsApps/Arch.exe)
- icon C:\Users\jacko\AppData\Local\wsltty\wsl.ico
- root C:\Users\jacko\scoop\persist\archwsl\data/rootfs

C:\Users\jacko\AppData\Local\wsltty>del "/F" "C:\Users\jacko\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\WSL Terminal.lnk"
Could Not Find C:\/F

C:\Users\jacko\AppData\Local\wsltty>del "/F" "C:\Users\jacko\AppData\Local\Microsoft\WindowsApps\WSL.bat"
Could Not Find C:\/F

C:\Users\jacko\AppData\Local\wsltty>del "/F" "C:\Users\jacko\AppData\Local\Microsoft\WindowsApps\WSL~.bat"
Could Not Find C:\/F

C:\Users\jacko\AppData\Local\wsltty>del "/F" "C:\Users\jacko\OneDrive\Desktop\WSL Terminal.lnk"
Could Not Find C:\/F
distro ''
- name 'WSL'
- guid DefaultVersion
- (launcher C:\Users\jacko\AppData\Local/Microsoft/WindowsApps/Arch.exe)
- icon C:\Users\jacko\AppData\Local\wsltty\wsl.ico
- root C:\Users\jacko\scoop\persist\archwsl\data/rootfs
Installing wsltty into C:\Users\jacko\AppData\Local\wsltty
```

**Comment by JackofSpades707** [comment]
Reinstalling results in the same issue (when installing the latest version 3.4.3)

Reinstalling version 3.0.6 fixes the issue but wsltty still exists on my system without a proper way to remove it.

**Comment by JackofSpades707** [comment]
It's very possible I tried to upgrade to a later version previously, noticed wsltty was messed up afterwards, and then reinstalled 3.0.6 (since I kept the original installer) so that may be extremely relevant information regarding this issue.

**Comment by JackofSpades707** [comment]
I'd be really grateful for a method/advice on how to completely remove wsltty (That way I can simply install it with scoop and depend on that to manage updating/uninstalling/ect)

**Comment by mintty** [comment]
Did you check instruction at https://github.com/mintty/wsltty#uninstallation? (cf #258)
Fully automatic uninstallation is currently not provided. (cf https://github.com/mintty/wsltty/issues/15#issuecomment-260767085)

**Comment by JackofSpades707** [comment]
> Did you check instruction at https://github.com/mintty/wsltty#uninstallation? (cf #258)
> Fully automatic uninstallation is currently not provided. (cf [#15 (comment)](https://github.com/mintty/wsltty/issues/15#issuecomment-260767085))

The only part I didn't follow was manually deleting the install directory. Considering the output, I thought it was safe to assume the uninstall script didn't do it's job properly so I didn't want to delete the directory just yet. (I suspected something similar to #15 might happen to me)

I may have run the uninstall script outside of the wsltty install directory after realizing that upgrading appeared to mess things up. As well, I can't quite remember for sure.

btw, thanks for making wsltty.

**Comment by mintty** [comment]
Released 3.4.4.
